manifests/database_server/firewall.pp

```diff
@@ -20,10 +20,10 @@
   $sources.each | $location, $source |
   {
     firewall { "200 allow OpenAFS via ${proto} from ${source}":
-      proto    => $proto,
-      dport    => $dports,
-      source   => $source,
-      action   => 'accept',
+      proto  => $proto,
+      dport  => $dports,
+      source => $source,
+      action => 'accept',
```

Comment:
This was just PDK applying some formatting/alignment changes introduced from a previous PR.